### DISCUSSION TITLE
About Startup time

### DISCUSSION THREAD

**Discussion by akuz0** [hoppscotch/hoppscotch#3936] [Q&A] [answered]
Hello everyone, can you please help me with the test results from the quick launch app (as described in the article)?
https://hoppscotch.com/blog/introducing-hoppscotch-desktop-application

My result: 5.1513
My environment: Windows10, intel core i5-10210u

My case:
Appliation run:
 C:\Program Files\Google\Chrome\Application\chrome_proxy.exe"  --profile-directory=Default --app-id=ajamoobebmdhjjmdelngengkpklcimmg
 Window Name:
 Hoppscotch • Open source API development ecosystem • Hoppscotch
 
![image](https://github.com/hoppscotch/hoppscotch/assets/91601482/084229a0-2519-4487-8839-773f9b5dfa3b)

**Comment by thetronjohnson** [comment, marked as answer]
Hi @akuz0 are you using the PWA?
The blog is regarding the native desktop app --> https://hoppscotch.com/download
